src/prefixer/RopmModule.ts

```diff
@@ -316,16 +316,14 @@ export class RopmModule {
         const prefixMapKeysLower = prefixMapKeys.map(x => x.toLowerCase());
 
         /**
-         * Get the alias for a namespace. Only returns if it exists and is different than what is given.
+         * Get the alias for a namespace.
          */
         const getAlias = (namespace?: string) => {
             if (namespace) {
                 const lowerNamespaceName = namespace.toLowerCase();
                 const idx = prefixMapKeysLower.indexOf(lowerNamespaceName);
-                const prefix = this.prefixMap[prefixMapKeys[idx]];
-                if (prefix && prefix.toLowerCase() !== lowerNamespaceName) {
-                    return prefix;
-                }
+
+                return this.prefixMap[prefixMapKeys[idx]];
```

Comment:
This is required now because of not adding a version number for packages with just one major version - there are situations when a prefix is the same as namespace and it doesn't mean this is an internal function/class